src/mixin/java/com/gtnewhorizons/angelica/mixins/early/angelica/textures/ic2/MixinRenderLiquidCell.java

```diff
@@ -15,13 +16,19 @@
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
 import org.spongepowered.asm.mixin.injection.callback.LocalCapture;
 
-@Mixin(RenderLiquidCell.class)
+@Mixin(value = RenderLiquidCell.class, remap = false)
```

Comment:
This makes it non obvious failure if someone adds a new mixin that does need a remap -- any reason not to add it on the individual methods?